src/common/SegmentedButton.tsx

```diff
@@ -0,0 +1,127 @@
+                <FontAwesomeIcon icon={faCertificate} />
+              ) : option.updates == Infinity ? (
+                "∞"
+              ) : (
+                option.updates
+              )}
+            </div>
+          )}
+          <ActionLink
+            link={{ 
+              onClick: () => {
+                setSelected(option.id);
+                if (option.onClick) {
+                  option.onClick();
+                }
+              },
+              href: option.href,
+            }}
+            className="segmented-button-link"
+            allowDefault={!!option.href && !option.onClick}
+          >
+            {option.label}
+          </ActionLink>
+        </div>
+      ))}
+      <style jsx>{`
+        .segmented-button {
+          border-radius: 25px;
+          background-color: ${REVERSE_THEME_COLOR};
+          border: 2px solid ${props.color || THEME_COLOR};
+          display: flex;
+          justify-content: space-evenly;
+        }
+        .segmented-button-option {
+          display: inline-block;
+          position: relative;
+        }
+        .segmented-button-option > :global(button),
+        .segmented-button-option > :global(a) {
+          border-radius: 25px;
+          padding: 8px 12px;
+          background-image: none;
+          color: ${props.color || THEME_COLOR};
+          background-color: ${REVERSE_THEME_COLOR};
+          border: 2px solid ${REVERSE_THEME_COLOR};
+          transition: all 0.2s linear 0s;
+          box-sizing: border-box;
+          display: inline-block;
+        }
+        .segmented-button-option > :global(button):focus,
+        .segmented-button-option > :global(a):focus {
+          outline: none;
+        }
+        .segmented-button-option:not(.selected):hover > :global(button),
+        .segmented-button-option:not(.selected):hover > :global(a) {
+          background-color: ${Color(props.color || THEME_COLOR).alpha(0.5)};
+          cursor: pointer;
+        }
+        .segmented-button-option.selected:hover > :global(button),
+        .segmented-button-option.selected:hover > :global(a) {
+          cursor: pointer;
+        }
+        .selected > :global(button),
+        .selected > :global(a) {
+          border: 2px solid ${REVERSE_THEME_COLOR};
+          color: ${REVERSE_THEME_COLOR};
+          background-color: ${props.color || THEME_COLOR};
+        }
+        .updates {
+          background-color: ${props.color || THEME_COLOR};
+          color: ${transparent ? "black" : REVERSE_THEME_COLOR};
+          position: absolute;
+          border-radius: 50%;
+          width: 20px;
+          height: 20px;
+          right: -5px;
+          top: -5px;
+          text-align: center;
+          font-size: 14px;
+          line-height: 20px;
+          font-weight: bold;
+          z-index: 2;
+          transition: all 0.2s linear 0s;
+        }
+        .selected .updates {
+          background-color: ${REVERSE_THEME_COLOR};
```

Comment:
Resident designer suggested that, rather than changing background color when selected, the updates should simply always have a `2px solid REVERSE_THEME_COLOR` border around them at all times.